packages/core/src/Stores/client-store.js

```diff
@@ -512,6 +513,10 @@ export default class ClientStore extends BaseStore {
         );
     }
 
+    get is_wallet_account() {
+        return this.account_list.some(account => account.loginid.includes('CRW') || account.loginid.includes('VRW'));
```

Comment:
@suisin-deriv we need to check for MFW as well as a possibility

Reply:
will check this as well @shahzaib-deriv 🙏